### DISCUSSION TITLE
Is Cs2-modded-server works with linuxgsm

### DISCUSSION THREAD

**Discussion by zatmania** [kus/cs2-modded-server#111] [General]
Hi,

i try to install a dedicated server and i find your github ! Thanks for your work
Is your modded server works with a linuxgsm install ? (linuxgsm.com)

Regards,

Zat

**Comment by mavproductions** [comment]
Hi @zatmania,

You can download the latest releases and/or code files and drag and drop the file/folder structure without the scripts and you should be fine.

There might also be a way to incorporate Linux GSM with the docker file included in the repo.

Any reason you want to stay with LinuxGSM? Multiple game support?

If you're only using Linux GSM for CS2, you should honestly consider switching to the Linux install script, or try your luck with docker.

Happy to help if you have any other questions just tag me.

**Reply by kus** [reply]
As @mavproductions said; if you are only using LinuxGSM for CS2 you don't need it if you use this repo.

This repo does the same as LinuxGSM where it installs packages required for Unbuntu/Debian and handles setting up SteamdCMD and downloading CS2 etc but then comes pre setup with all the mods configured.

[These](https://github.com/kus/cs2-modded-server?tab=readme-ov-file#running-on-linux) are the handful of steps required getting setup on a brand new Ubuntu/Debian instance.

**Comment by zatmania** [comment]
Hi,

thanks for your response. Many Thanks to your work !
I will test docker version whan i have some times to do this :)
